### ISSUE TITLE
code

### ISSUE THREAD

**Issue by Zsynapsis** [mjq11302010044/RRPN_plusplus#18]
Hello author, 
    I have a few questions about the code. 
    First, look at the example you gave using the yaml file isconfigs/arpn/e2e_rrpn_R_50_C4_1x_train_AFPN_RT_LERB.yaml，the META ARCHITECTURE is ARPN. why not use RRPN? 
    Second, the results I got didn't feel very good, (see picture below) I want to ask why? 
Third, after I tried to train RRPN using the configs/rrpn/e2e_rrpn_R_50_C4_1x_ICDAR13_15_trial.yaml model, the loss=nan situation occurred. I reduced the learning rate, but it didn't work. 
    Finally, we look forward to hearing from you，thanks.

**Comment by Zsynapsis** [comment]
![2022-05-27 11-22-43 的屏幕截图](https://user-images.githubusercontent.com/73429498/170622865-74b11268-69cc-4fb8-a9c6-1619660f96d9.png)
Dear author, the problem is shown in the figure. How to solve it?

**Comment by mjq11302010044** [comment]
@Yidi-2896 In our repo, we only ensure the validity of the proposed RRPN++ (ARPN). Please see the tech report for more details.

**Comment by Zsynapsis** [comment]
Thanks for your answer!
